app/javascript/react/components/WelcomeTile.test.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import Enzyme, { mount } from "enzyme"
+import Adapter from "enzyme-adapter-react-16"
+import { BrowserRouter } from 'react-router-dom'
+Enzyme.configure({ adapter: new Adapter() })
+
+import WelcomeTile from "./animals/WelcomeTile"
+
+describe("WelcomeTile", () => {
+  let wrapper, onClickMock
+
+  beforeEach(() => {
+    onClickMock = jest.fn()
+    wrapper = mount(
+      <BrowserRouter>
+        <WelcomeTile />
+      </BrowserRouter>
+    )
+  })
+
+  it("renders an h1 tag with the animal name", () => {
```

Comment:
Small thing, but this test description doesn't match the expectation of an h1 title.